src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "FV3 Forecast config",
+  "description": "This document is to validate user-defined FV3 forecast config files",
+  "type": "object",
+  "properties": {
+    "platform": {
+      "description": "attributes of the platform",
+      "type": "object",
+      "properties": {
+        "WORKFLOW_MANAGER": {
+          "type": "string",
+          "enum": [
+            "rocoto",
+            "none"
+          ]
+        },
+        "NCORES_PER_NODE": {
+          "type": "number"
+        },
+        "SCHED": {
+          "type": "string",
+          "enum": [
+            "slurm",
+            "pbspro",
+            "lsf",
+            "lsfcray",
+            "none"
+          ]
+        },
+        "CCPA_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MRMS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "NDAS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "METPLUS_PATH": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MET_BIN_EXEC": {
+          "type": "string"
+        },
+        "MET_INSTALL_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "DOMAIN_PREGEN_BASEDIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "PARTITION_DEFAULT": {
+          "type": "string"
+        },
+        "QUEUE_DEFAULT": {
+          "type": "string"
+        },
+        "PARTITION_FCST": {
+          "type": "string"
+        },
+        "QUEUE_FCST": {
+          "type": "string"
+        },
+        "PARTITION_HPSS": {
+          "type": "string"
+        },
+        "QUEUE_HPSS": {
+          "type": "string"
+        }
+      }
+    },
+    "parameters": {
+      "description": "parameters of the forecast",
+      "type": "object",
+      "properties": {
+        "LENGTH": {
+          "type": "string",
+          "format": "uri"
+        },
+        "GRID": {
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "run_cmd": {
+      "description": "parameters for the run command",
+      "type": "object",
```

Comment:
Does it make sense for us to use all caps here, or match the required variable names in the FV3Forecast method?